dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -97,17 +105,20 @@ public boolean equals(Object obj) {
             return false;
         }
         final WorkflowVersion other = (WorkflowVersion)obj;
-        return Objects.equals(super.getName(), other.getName()) && Objects.equals(super.getReference(), other.getReference());
+        return Objects.equals(super.getName(), other.getName()) && Objects.equals(super.getReference(), other.getReference())
```

Comment:
It was like this, but it's confusing to me why it's calling `super.getName()` and `super.getReference`, because

1. This class doesn't override those methods
2. If it did override those methods, you would most likely want to compare this class' result, not the super class.

Reply:
Might be a mistake